packages/wallets/src/evm/utils/analytics.ts

```diff
@@ -0,0 +1,28 @@
+import { fetch } from "cross-fetch";
```

Comment:
don't need `cross-fetch` anymore we no longer support any environment that doesn't have native fetch